test_files/expected/tester_config_nillable.yaml

```diff
@@ -0,0 +1,18 @@
+nillable:
+  pds:ASCII_Integer:
+    inapplicable: -9999
+    missing: -9988
+    unknown: -9977
+    anticipated: -9966
+
+  pds:ASCII_Real:
+    inapplicable: -9999.0
+    missing: -9988.0
+    unknown: -9977.0
+    anticipated: -9966.0
+
+  pds:ASCII_Short_String_Collapsed:
+    inapplicable: inapplicable_alt
+    missing: missing_alt
+    unknown: unknown_alt
+    anticipated: anticipated_alt
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/SETI/rms-pds4indextools/pull/38#issuecomment-2310596918>#91ec65</a><span> - 08/26/2024, 04:41 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The configuration uses custom values for special constants instead of PDS4 standard values, which may lead to inconsistencies with PDS4 specifications.
</div>
<div id="fix">
<b>Fix:</b> Update the special constant values to align with PDS4 standards. Use 'N/A' for not applicable, 'UNK' for unknown, and 'null' for intentionally blank values.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -1,18 +1,18 @@
  nillable:
    pds:ASCII_Integer:
 -    inapplicable: -9999
 -    missing: -9988
 -    unknown: -9977
 -    anticipated: -9966
 +    inapplicable: N/A
 +    missing: null
 +    unknown: UNK
 +    anticipated: null
 
    pds:ASCII_Real:
 -    inapplicable: -9999.0
 -    missing: -9988.0
 -    unknown: -9977.0
 -    anticipated: -9966.0
 +    inapplicable: N/A
 +    missing: null
 +    unknown: UNK
 +    anticipated: null
 
    pds:ASCII_Short_String_Collapsed:
 -    inapplicable: inapplicable_alt
 -    missing: missing_alt
 -    unknown: unknown_alt
 -    anticipated: anticipated_alt
 +    inapplicable: N/A
 +    missing: null
 +    unknown: UNK
 +    anticipated: null
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged

Reply:
These are alternate, "user-defined" values.